var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -95,12 +100,25 @@ def configure_args(self):
         options = []
         oapp = options.append
 
+        if '+wannier90' in spec:
+            oapp('--with-wannier90-libs=-L{0}'
+                 .format(spec['wannier90'].prefix.lib + ' -lwannier -lm'))
+            oapp('--with-wannier90-incs=-I{0}'
+                 .format(spec['wannier90'].prefix.modules))
+            oapp('--with-wannier90-bins={0}'
+                 .format(spec['wannier90'].prefix.bin))
+            oapp('MPIFC={0}/mpifc'.format(spec['mpi'].prefix.bin))
+            oapp('--enable-connectors')
+            oapp('--with-dft-flavor=wannier90')
+
         if '+mpi' in spec:
             # MPI version:
             # let the configure script auto-detect MPI support from mpi_prefix
             oapp('--with-mpi-prefix={0}'.format(spec['mpi'].prefix))
             oapp('--enable-mpi=yes')
             oapp('--enable-mpi-io=yes')
+            if '~wannier90' in spec:
```

Comment:
I don't think this should be restricted to when mpi is enabled. Shoudn't this be moved to an `else` block closing the `if +wannier90` in spec:` conditional above?